src/CTA.WebForms2Blazor/FileConverters/StaticFileConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CTA.WebForms2Blazor.FileInformationModel;
+
+namespace CTA.WebForms2Blazor.FileConverters
+{
+    public class StaticFileConverter : FileConverter
+    {
+        public StaticFileConverter(string relativePath) : base(relativePath)
+        {
+
+        }
+
+        public override async Task<IEnumerable<FileInformation>> MigrateFileAsync()
+        {
+            var newPath = Path.Combine("wwwroot", RelativePath);
+            var workingDirectory = Environment.CurrentDirectory;
+            var testProjectPath = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
```

Comment:
Also concerned about jon's comment here but this appears to be the context prefix for the path rather than actual test project path, perhaps just a naming oversight?

Reply:
Yep, I'll rename it!